src/qt/veil/transactiondetaildialog.cpp

```diff
@@ -39,8 +40,13 @@ TransactionDetailDialog::TransactionDetailDialog(QWidget *parent, TransactionRec
     ui->textSize->setProperty("cssClass" , "text-detail");
     ui->textStatus->setProperty("cssClass" , "text-detail");
     connect(ui->btnEsc,SIGNAL(clicked()),this, SLOT(onEscapeClicked()));
+    connect(ui->explorerButton,SIGNAL(clicked()),this, SLOT(onExplorerClicked()));
+
 
     if(rec) {
+        QString baseStr = "https://explorer.veil-project.com/tx/";
```

Comment:
Shouldn't this address be chain dependent?

Reply:
Good suggestion. @drag0ng the testnet explorer is: https://testnet.veil-project.com